tajo-rpc/src/test/java/org/apache/tajo/rpc/TestBlockingRpc.java

```diff
@@ -252,8 +257,11 @@ public void testConnectionFailed() throws Exception {
     
     try {
       int port = server.getListenAddress().getPort() + 1;
-      client = new BlockingRpcClient(DummyProtocol.class,
-          RpcUtils.getConnectAddress(new InetSocketAddress("127.0.0.1", port)), retries);
+      RpcConnectionPool.RpcConnectionKey rpcConnectionKey =
+          new RpcConnectionPool.RpcConnectionKey(
+              RpcUtils.getConnectAddress(new InetSocketAddress("127.0.0.1", port)),
+              DummyProtocol.class, false);
+      client = new BlockingRpcClient(rpcConnectionKey, retries);
```

Comment:
This code block does not throw ConnectException. how about add acquire connection on this?